sdk/endpoint/compositeBuilder.go

```diff
@@ -0,0 +1,29 @@
+package endpoint
```

Comment:
This one is good. I don't mind the name of the file, but we usually do not use camel case. I wonder how the linter passes here.

Reply:
thanks, fixed